Gemfile

```diff
@@ -73,7 +71,7 @@ group :test do
 end
 
 group :production do
-  gem 'exception_notification', '~> 4.0.0'
+  gem 'exception_notification', '~> 4.2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.